### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by PaulTheFat** [keithlau2015/Final-Year-Project_-MazeChallenger#10]
Post any bugs you found in development here

Known Bug:
Player running problem: sometimes player can't run when pressed shift
map overlap situation: rooms are spawning into each other
player attack sound effect / walking sound effect / jumping sound effect: sound delays

Known issues:
Map generator not stable: time spent generating is too long

**Comment by PaulTheFat** [comment]
if the level builder always use L room the waiting time will be longer

**Comment by PaulTheFat** [comment]
Map loading time: 10 seconds a bit long, 20 seconds is unacceptable

**Comment by keithlau2015** [comment]
> if the level builder always use L room the waiting time will be longer

Not really, after I test few more times, the L room will not affect the waiting time, is the mesh side problem

**Comment by keithlau2015** [comment]
resolution problem: need to resize the 3D objects in different resolution, or else some of them can't show in the screen
